docs/javascript/conjur-account.js

```diff
@@ -56,7 +56,13 @@ function displayError(formField, message) {
 }
 
 $(document).ready(function() {
-  var accountCookie = getAccountCookie();
+  var ipAddress = null;
+
+  $.getJSON("http://ipv4.jsonip.com/", function (data) {
```

Comment:
This feels like it could be problematic. Are we ok with the IP address not getting passed to the form (e.g. because this requests fails or is really slow)?

Is there a reason not to use the client's IP address that we see on the server?

Reply:
Since we're making a request from Conjur to CPanel, isn't the IP address of the request going to be for the Conjur host?

Brian (Sterling) says we can do without the IP address, so I'll just remove this.

Reply:
I'm not sure what you mean about making the request from Conjur to CPanel. 

The form on this page gets submitted to CPanel. The client's address will be available to that entrypoint. It could fill in the IP address as part of processing the request.

Having said that, though, if we don't need the IP address, it's probably better to get rid of it altogether.

Reply:
Sounds good, removed.